utils/build/build.sh

```diff
@@ -283,15 +283,15 @@ TEST_LIBRARY="${TEST_LIBRARY:-${DEFAULT_TEST_LIBRARY}}"
 BINARY_PATH="${BINARY_PATH:-}"
 BINARY_URL="${BINARY_URL:-}"
 
-if [[ ! -d "${SCRIPT_DIR}/docker/${TEST_LIBRARY}" ]]; then
+if [[ "${BUILD_IMAGES}" =~ /weblog/ && ! -d "${SCRIPT_DIR}/docker/${TEST_LIBRARY}" ]]; then
```

Comment:
This check was preventing `build.sh` to proceed but it's irrelevant when building `runner` and `agent`.

Indeed there seems to be much assumptions in the code flow of this script that it's building `weblog` images, even when it is not.